Document new phone number & email formatting (#391)
discord-userdoccers/discord-userdoccers: pages/reference.mdx

| User ^1^                | `<@!USER_ID>`                  | `<@!80351110224678912>`                     |
| Channel                 | `<#CHANNEL_ID>`                | `<#103735883630395392>`                     |
| Role                    | `<@&ROLE_ID>`                  | `<@&165511591545143296>`                    |
| Slash Command ^2^       | `</NAME:COMMAND_ID>`           | `</airhorn:816437322781949972>`             |
| Standard Emoji          | Unicode Characters or `:NAME:` | 💯 or `:100:`                               |
| Custom Emoji            | `<:NAME:ID>`                   | `<:mmLol:216154654256398347>`               |
| Custom Emoji (Animated) | `<a:NAME:ID>`                  | `<a:b1nzy:392938283556143104>`              |
| Unix Timestamp          | `<t:TIMESTAMP>`                | `<t:1618953630>`                            |
| Unix Timestamp (Styled) | `<t:TIMESTAMP:STYLE>`          | `<t:1618953630:d>`                          |
| Guild Navigation        | `<id:TYPE>`                    | `<id:customize>`                            |
| Email ^3^               | `<USERNAME@DOMAIN>`            | `<nelly@discordapp.com>`                    |

> | Phone Number ^4^        | `<+PHONE_NUMBER>`              | `<+1 (555) 123 4567>` or `<tel:5551234567>` |

Reviewer: ```suggestion
| Phone Number ^4^        | `<+PHONE_NUMBER>`              | `<+1 (555) 123 4567>` |
```
no point here

| Unix Timestamp          | `<t:TIMESTAMP>`                | `<t:1618953630>`                |
| Unix Timestamp (Styled) | `<t:TIMESTAMP:STYLE>`          | `<t:1618953630:d>`              |
| Guild Navigation        | `<id:TYPE>`                    | `<id:customize>`                |
| Email ^3^               | `<USERNAME@DOMAIN>`            | `<nelly@discordapp.com>`        |

> | Phone Number ^3^        | `<+PHONE_NUMBER>`              | `<+1 (555) 123 4567>`           |

Reviewer 2: ```suggestion
| Phone Number ^4^        | `<+PHONE_NUMBER>`              | `<+1 (555) 123 4567>`           |
```

> | Phone Number ^4^        | `<+PHONE_NUMBER>`              | `<+1 (555) 123 4567>`           |

Reviewer: unnecessary imo

^2^ Subcommands and subcommand groups can also be mentioned by using respectively `</NAME SUBCOMMAND:ID>` and `</NAME SUBCOMMAND_GROUP SUBCOMMAND:ID>`.

> ^3^ Can be optionally prefixed with `mailto:`, following the same format as the [`mailto:`](https://en.wikipedia.org/wiki/Mailto) URI scheme. Does not support multiple comma separated addresses.

Reviewer: ```suggestion
^3^ Can be optionally prefixed with `mailto:`, following the same format as the [`mailto:`](https://en.wikipedia.org/wiki/Mailto) URI scheme. Does not support multiple comma-separated addresses. Supports headers (e.g. `<nelly@discord.com?subject=Title&body=Message%20Content>`).
```

^2^ Subcommands and subcommand groups can also be mentioned by using respectively `</NAME SUBCOMMAND:ID>` and `</NAME SUBCOMMAND_GROUP SUBCOMMAND:ID>`.

> ^3^ Email and phone number markdown uses `mailto:` and `tel:` URI schemes respectively that can optionally be prefixed (e.g. `<mailto:nelly@discordapp.com>`). Email markdown supports headers, values must be [URL Encoded](https://en.wikipedia.org/wiki/Percent-encoding) (e.g. `<nelly@discordapp.com?subject=Message%20Title&body=Message%20Content>`).

Reviewer 2: ```suggestion
^3^ Can be optionally prefixed with `mailto:`, following the same format as the [`mailto:`](https://en.wikipedia.org/wiki/Mailto) URI scheme. This doesn't support multiple comma separated addresses.

^4^ Can be optionally prefixed with `tel:`/`sms:`, the `+` is optional when specifying a scheme and it doesn't adhere to the `tel:` URI scheme specification. Specifying headers with `sms:` is not supported
```
